### ISSUE TITLE
High accuracy and low loss in training but low accuracy and high loss when validation

### ISSUE THREAD

**Issue by s2244521** [calmisential/Basic_CNNs_TensorFlow2.0#36]
My validation data is the same as training data but the validation result is not good as training in training process. Anyone met this issue?

**Comment by s2244521** [comment]
I think BN, dropout adn augmentation are the reasons.

**Comment by s2244521** [comment]
solved
